src/FFTW.jl

```diff
@@ -11,6 +11,8 @@ import AbstractFFTs: Plan, ScaledPlan,
                      fftshift, ifftshift,
                      rfft_output_size, brfft_output_size,
                      plan_inv, normalization
+import FFTW_jll
+import MKL_jll
```

Comment:
This will unconditionally force installation of MKL in all cases.

Reply:
Ah I was worried something like this would happen 😅

Reply:
So installation only happens when importing a jll library? Isn't it installed anyway since it's a package dependency? Or is this a MKL-specific thing?

Reply:
MKL_jll is lazy, which means it's installed only on `using`/`import`.